src/datasource.test.ts

```diff
@@ -61,61 +56,76 @@ describe('AthenaDatasource', () => {
       jsonData: { defaultRegion: 'testRegion', catalog: 'testCatalog', database: 'testDatabase' },
     } as unknown as DataSourceInstanceSettings<AthenaDataSourceOptions>;
     ctx.ds = new DataSource(ctx.instanceSettings);
-    runtime.setBackendSrv(backendSrv as runtime.BackendSrv);
+    ctx.ds.getResource = jest.fn().mockImplementation((path: string) => {
```

Comment:
why were these changes needed is setBackendSrv not a thing in the newer version? Looks good tho, just curious!

Reply:
`setBackendSrv` is still a thing, but after upgrading, there were errors with needing to mock `fetch` from the `BackendSrv`. i didn't want to spend time mocking out the internals of the `BackendSrv`.